.github/dependabot.yml

```diff
@@ -22,6 +22,3 @@ updates:
       - "gradle"
     commit-message:
       prefix: "[PF-2983]"
-    ignore:
-      # PF-3000: k8s client versions 20.0.0 onward without '-legacy' suffix include breaking changes
-      - dependency-name: "io.kubernetes:client-java"
```

Comment:
I think this is okay to take out in this PR, but flagging that we might ultimately add it back in if k8s client keeps on publishing legacy and non-legacy versions.  If they do, Dependabot will pick up on the last published version regardless of its legacy status.  I don't think there's a way to tell Dependabot to allow or exclude versions adhering to a naming convention (outside of the semver major, minor, patch designations).